src/doblib/module.py

```diff
@@ -67,6 +67,18 @@ def _run_migration(self, db_name, script_name):
             )
             script.migrate(env, version)
 
+    def _run_migration_sql(self, db_name, script_name):
+        """Run a migration SQL script by executing the migrate function"""
+        if not os.path.isfile(script_name):
+            return
+
+        with open(script_name, "r") as f:
+            sql = f.read()
+
+        utils.info(f"Executing {script_name} script")
+        with self.env(db_name) as env:
+            env.cr.execute(sql)
```

Comment:
Going over `env(...)` will initialize Odoo and cause the same problems again where the database scheme doesn't match the python models. We only need the cursor at this point:

```
with closing(db.cursor()) as cr, open(script_name, "r") as f:
    cr.execute(f.read())
```
Should be enough here.

On execution an info log would be nice (see the `_run_migration`).